### PR TITLE
imessage-ruby 0.2.0

### PR DESCRIPTION
**Pull request by linjunpop**
https://github.com/linjunpop/imessage

### CODE REVIEW

**Comment by mikemcquaid** [comment]
We might need a less generic name here.

File: Library/Formula/imessage.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class Imessage < Formula
+  desc "Command-line tool to send iMessage"
+  homepage "https://github.com/linjunpop/imessage"
+  url "https://github.com/linjunpop/imessage/archive/v0.2.0.tar.gz"
+  version "0.2.0"
```
Comment (bfontaine) [inline Library/Formula/imessage.rb]: You can probably drop this line.

**Comment by DomT4** [comment]
Failed on ML. Possibly using unsupported syntax upstream for Ruby 1.8?

**Comment by linjunpop** [comment]
@mikemcquaid Maybe I change the formula's name to `imessage-ruby`, is that acceptable?

**Comment by linjunpop** [comment]
@DomT4 it seems 1.8 is EOL. Can I drop support for ML?

**Comment by mikemcquaid** [comment]
@linjunpop Sounds good. Up to you if you drop 1.8 support.

File: Library/Formula/imessage-ruby.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+class ImessageRuby < Formula
+  desc "Command-line tool to send iMessage"
+  homepage "https://github.com/linjunpop/imessage"
+  url "https://github.com/linjunpop/imessage/archive/v0.2.0.tar.gz"
+  sha256 "5d67df55a53d0fa657ea439c240a3b322dd0b83a102fa881915569f44431eb00"
+  head "https://github.com/linjunpop/imessage.git"
+
+  def install
+    if MacOS.version <= :mountain_lion
```
Comment (DomT4) [inline Library/Formula/imessage-ruby.rb]: We have syntax for this already. You want `depends_on :macos => :mavericks` one empty line under the `head` line.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/c64a4d2a6bb5c204340370881f666c638e496fa1. Thank you for your contribution to Homebrew, we appreciate it! :smiley_cat:

**Pull request closed without merging**